File: src/nl/rubensten/texifyidea/action/insert/InsertTypewriterAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertTypewriterAction extends InsertEditorAction {

    public InsertTypewriterAction() {
        super("Typewriter (monospace)", TexifyIcons.FONT_TYPEWRITER, "\\texttt{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertTypewriterAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertTypewriterAction extends InsertEditorAction {

    public InsertTypewriterAction() {
        super("Typewriter (monospace)", TexifyIcons.FONT_TYPEWRITER, "\\texttt{", "}");
    }
}
